conf/nginx.conf

```diff
@@ -137,7 +137,7 @@ location ^~ __PATH__/ {
     access_log off;
   }
 
-  location ~ ^__PATH__/.+[^\/]\.(?:png|html|ttf|ico|jpg|jpeg|bcmap)$ {
+  location ~* \.(?:png|html|ttf|ico|jpg|jpeg|bcmap|mp4|webm)$ {
```

Comment:
Could you explain why you need this change ? I remeber there was something around this on "15 to 18" PR

Reply:
Here it's just the application of the [official recommendation](https://docs.nextcloud.com/server/19/admin_manual/installation/nginx.html)

Reply:
It's a good addition, on the onboarding slides when you first access Nextcloud with your user, if you dont add `mp4|webm` locations you won't be able to play the video from the first slide. It's just an example of use.

Reply:
https://docs.nextcloud.com/server/19/admin_manual/installation/nginx.html#nextcloud-in-a-subdir-of-the-nginx-webroot

IMO, we should  probably keep `~ ^__PATH__/.+[^\/]\.`